wpilibc/src/main/native/cpp/PneumaticHub.cpp

```diff
@@ -93,17 +107,51 @@ void PneumaticHub::EnableCompressorDigital() {
   FRC_ReportError(status, "Module {}", m_module);
 }
 
-void PneumaticHub::EnableCompressorAnalog(units::volt_t minAnalogVoltage,
-                                          units::volt_t maxAnalogVoltage) {
+void PneumaticHub::EnableCompressorAnalog(
+    units::pounds_per_square_inch_t minPressure,
+    units::pounds_per_square_inch_t maxPressure) {
+  if (minPressure.value() >= maxPressure.value()) {
+    throw FRC_MakeError(err::InvalidParameter, "{}",
+                        "maxPressure must be greater than minPresure");
+  }
+  if (minPressure.value() < 0 || minPressure.value() > 120) {
+    throw FRC_MakeError(err::ParameterOutOfRange,
+                        "minPressure must be between 0 and 120 PSI, got {}",
+                        minPressure.value());
```

Comment:
If you include <frc/fmt/Units.h>, you shouldn't have to use `.value()`, and the unit suffix will be included in the stringification.